rango/views.py

```diff
@@ -25,91 +22,69 @@ def index(request):
     visitor_cookie_handler(request)
     context_dict["visits"] = request.session["visits"]
 
+    # import ipdb; ipdb.set_trace()
```

Comment:
Delete unused code. Such code should not be presented in the project when you send it to the PR.